package.json

```diff
@@ -16,7 +16,8 @@
     "prebuild:openstack": "rm -rf dist/openstack",
     "build:openstack": "webpack --config webpack.openstack.prod.js",
     "build-n-zip:oVirt": "yarn build:oVirt && yarn zip:oVirt",
-    "build-n-zip:openstack": "yarn build:openstack && yarn zip:openstack"
+    "build-n-zip:openstack": "yarn build:openstack && yarn zip:openstack",
+    "build-n-zip:openstack-and-oVirt": "yarn build:oVirt && yarn zip:oVirt && yarn build:openstack && yarn zip:openstack"
```

Comment:
`build-n-zip` or `build-n-zip:packages` i think is a better name. 
then:
`"yarn build-n-zip:oVirt && yarn build-n-zip:openstack"`